src/main/java/com/example/TimeBrew_Server/domain/Time.java

```diff
@@ -0,0 +1,34 @@
+package com.example.TimeBrew_Server.domain;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Table(name = "Time")
+@Getter
+@Setter
+public class Time {
+
+    @Id
```

Comment:
이 애너테이션이 잘못된 것 같아요.
```suggestion
    @Table(name = "Time")
```
으로 변경하면 좋겠네요!